GlobalStaticVersion.props

```diff
@@ -8,12 +8,12 @@
     <SemanticVersionMajor>2</SemanticVersionMajor>
     <SemanticVersionMinor>2</SemanticVersionMinor>
     <SemanticVersionPatch>0</SemanticVersionPatch>
-    <PreReleaseMilestone>beta6</PreReleaseMilestone>
+    <PreReleaseMilestone></PreReleaseMilestone>
     <!-- 
       Date when Semantic Version was changed. 
       Update for every public release.
     -->
-    <SemanticVersionDate>2016-06-02</SemanticVersionDate>
+    <SemanticVersionDate>2016-12-05</SemanticVersionDate>
```

Comment:
minor version stays the same. FileVersion will be something like `2.2.<number of 5 minutes intervals since SemanticVersionDate>` So resetting it now will generate file version that is less than for all `-beta` versions before.